feat: add example of version of schema in adeo.yml file

animeshk923 opened the pull request:
**Description**

- This PR fixes the second bullet point of #1615.
- Requesting review @derberg @akshatnema @magicmatatjahu 

**Related issue(s)**
Fixes #1615

animeshk923 committed:
Add example of version of schema in adeo.yml file

Changes made:
	- This PR fixes the second bullet point of #1615.

animeshk923 commented on config/casestudies/adeo.yml:
```diff
@@ -87,8 +87,21 @@ schemas:
+    matches the git tag version of the product. Every schema has a `version` information 
+    that matches with product tag version.
+
+    Example Avro schema with version information:
```
@derberg 
```Example Avro schema with version information:
    ```json
    {
      "namespace": "com.adeo.casestudy.costingrequest",
      "type": "record",
      "name": "CostingRequestPayload",
      "version": "1.1.0",
      "fields": [ ... ]
    }
```
Does this whole code goes inside the `versioning:` key?

derberg replied:
yup, well done

animeshk923 replied:
> yup, well done
- Thank you so much :bow: 

- Fixed `validation` with this commit a3fb042656d8a302f99a20f7ac21d2022e89a491

derberg reviewed (changes requested):
I think by accident you removed `validation: Based on validation using Confluent Schema Registry.` so please bring it back and also you can take it out of Draft so we can approve and merge

animeshk923 commented:
> I think by accident you removed validation: Based on validation using Confluent Schema Registry. so please bring it back and also you can take it out of Draft so we can approve and merge


Oops! :scream: Let me fix that...

animeshk923 committed:
Add example of version of schema in adeo.yml file

Previous changes:
- This PR fixes the second bullet point of #1615.

Changes in this commit:
- Fix the line containing `validation` on line 90
  that I accidently deleted. :(

animeshk923 commented:
Rendered correctly I guess @derberg  :smile: 
![Screenshot from 2023-07-13 23-17-39](https://github.com/asyncapi/website/assets/99868037/92276bd7-eb8e-4d29-a407-5cfccb757c93)

derberg reviewed (approved):
Thanks a 💯 it's looking great

derberg committed:
Merge branch 'master' into adeo-case-second-bullet-point

derberg commented:
/rtm

derberg commented:
@allcontributors please add @AnimeshKumar923 for docs,review

animeshk923 commented:
> Thanks a 100 it's looking great

Thank you so much :pray: :bow:
Feels great to help here. More to come from my side... :rocket:
